Monika After Story/game/script-topics.rpy

```diff
@@ -12850,6 +12850,103 @@ label monika_unknown:
         return "love"
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_brave_new_world",category=['literature'],prompt="Brave New World", random=True))
+
+label monika_brave_new_world:
+    m 3eub "There's another book I've been reading, [player]."
+    m 3eua "It's called 'Brave New World', it's another dystopian story. I wonder if you've heard of it?"
+    m 4eua "The idea is, you've got this futuristic world where humans are no longer born through natural means."
+    m 4eub "Instead, we are bred in hatcheries using test tubes and incubators, and engineered into castes from our conception."
+    m 1esb "Your future role in society would be decided beforehand, and you would be given a body and mind fitting of your predetermined purpose."
+    m 3esd "You would also be indoctrinated from birth to be satisfied with your lot in life, and not to seek anything different."
+    m 3esd "For example, people destined for manual labor would be designed to have limited cognitive capabilities."
+    m 1esc "They would be made to associate books to negative stimuli; so when they become adults, they would naturally tend to avoid reading."
+    m 4esd "They would also be taught to respect and submit to people from castes above theirs, and to look down on those of castes below."
+    m 2eua "It's a pretty interesting case as a dystopia, as most of them show you a crushed and oppressed people..."
+    m 3wuo "But in this one, everyone is actually happy, and genuinely supportive of the system!"
+    m 3esc "And despite that, to us, the readers, this still feels horrifying."
+    m 1rud "Sure, they managed to get rid of most of the human sufferings, or the fear of death..."
+    m 3esd "But it came at the price of getting people rid of any form of creativity and critical thinking."
+    m 2wud "We're talking about a world where you can get arrested just for reading poetry in public! Can you imagine that?"
+    m 3esc "One key point in the book is people not being able to appreciate old theatre plays..."
+    if seen_event("monika_pluralistic_ignorance"):
+        m 3hfu "Even if they are Shakespeare's plays, and you know how I feel about those..."
+    m 2esc "They just can't understand the value in the variety of human emotions, like sorrow, or loneliness."
+    m 4esc "They never experience those anymore. All of their desires are swiftly granted, and they never want for something they cannot get."
+    m 2rsc "..."
+    m 3rsc "And yet, despite all that, everyone is happy, healthy and safe..."
+    m 1esc "This really makes you think about the nature of happiness, and society..."
+    if mas_isMoniDis(lower=True):
+        m 2dsc "..."
+        m 2ekc "Sometimes, I wish I could fit in a world like that."
```

Comment:
The way i meant it is : She wish she could live happily in a world devoid of human emotions (e.g. the world of DDLC), and she can't since she went throught her epiphany.
So, i might be dumb, but do you mean you'd keep that sentence as is? I don't see any difference.

Reply:
I think I meant "be". 

Maybe it's just me, but I think fitting into something and being in something are a bit different. I don't think it's any bad as is, but that's what I think.

Reply:
Yes, I think fitting into something and being in something are different. Changinf "fit" by "be" completely change the meaning of the sentence, to me.
But if you think it sounds weird with 'fit in', you'll probably not be the only one in that case.
How about this, then?
"Sometimes, I wish I could live happily in a world like that."
"Maybe it was a bad thing I had my epiphany..."
"I could have kept on living without ever realizing the truth..."
"And I would not have to feel this pain..."

Reply:
Yeah I think that works and makes it clearer.